sv/socklog-unix/run

```diff
@@ -1,2 +1,7 @@
 #!/bin/sh
+
+if [ ! -d /var/log/socklog ] ; then
+  mkdir -m 2750 /var/log/socklog
+fi
+
```

Comment:
Just leave this to `log/run`. The main service doesn't actually rely on the existence of `/var/log/socklog`.

Reply:
@ahesford Thanks for the hint, DONE.